k8s-do/variables.tf

```diff
@@ -0,0 +1,43 @@
+# Set the variable value in *.tfvars file
+# or using -var="do_token=..." CLI option
+variable "do_token" {}
```

Comment:
I think that `do_token` should be of `string` type :)

Reply:
It is actually. Defaults to string always. Also followed the example at their official docs: https://registry.terraform.io/providers/digitalocean/digitalocean/latest/docs

Reply:
Thanks!